deb/debian/stackdriver-agent.init

```diff
@@ -68,7 +68,17 @@ find_libjvm_so () {
     echo $java_lib_dir
 }
 
-[ -n "$JAVA_LIB_DIR" ] || JAVA_LIB_DIR=$(find_libjvm_so)
+find_libjvm_so_jdk () {
+    local java_lib_dir
+    local java_home=$(dirname "$(readlink -f "$(bash -lc "which java 2>/dev/null")")")/..
+    if [ -f "$java_home/lib/server/libjvm.so" ]; then
+        java_lib_dir="$java_home/lib/server"
+    fi
+    echo $java_lib_dir
+}
+
+JAVA_LIB_DIR=$(find_libjvm_so)
```

Comment:
This will no longer allow users to override the auto-detected location by setting `JAVA_LIB_DIR` in `/etc/default/google-fluentd`.

Reply:
Indeed, I'll remove this part.